### ISSUE TITLE
Shuffle don't work at start

### ISSUE THREAD

**Issue by epicbenz** [joeyhage/homebridge-spotify-speaker#51]
When the Automation get started after a little time it always starts with the first song in the Playlist. After that the Shuffle works correctly. Also when the Automation gets activated in a short period of time it plays randomly songs of the Playlist. 

**Logs:**

```
[8/8/2023, 10:22:12 PM] [HomebridgeSpotifySpeaker] Restoring existing accessory from cache: Test
[8/9/2023, 1:10:14 AM] [Homebridge UI] Running job to cleanup config.json backup files older than 60 days...
[8/9/2023, 5:39:38 AM] [Homebridge UI] Running scheduled instance backup...
[8/9/2023, 5:39:38 AM] [Homebridge UI] Creating temporary backup archive at /tmp/homebridge-backup-8AiifT/homebridge-backup-0E13C5274D20.tar.gz
[8/9/2023, 11:50:19 AM] [TadoPlatform] Zuhause Wohnzimmer Heater: State: 0
[8/9/2023, 11:50:36 AM] [TadoPlatform] Zuhause Bad Heater: State: 0
[8/9/2023, 11:51:54 AM] [TadoPlatform] Zuhause Schlafzimmer Heater: State: 0
[8/9/2023, 1:03:34 PM] [TadoPlatform] Zuhause Wohnzimmer Heater: State: 3
[8/9/2023, 1:08:30 PM] [TadoPlatform] Zuhause Bad Heater: State: 3
[8/9/2023, 1:08:40 PM] [TadoPlatform] Zuhause Schlafzimmer Heater: State: 3
```

**Plugin Config:**

```json
{
    "bridge": {
        "name": "Homebridge 4XXX",
        "username": "0E:13:C5:xx:xx:20",
        "port": 51283,
        "pin": "xxx-xx-xxx",
        "advertiser": "avahi"
    },
    "accessories": [],
    "platforms": [
        {
            "name": "Config",
            "port": 8581,
            "platform": "config"
        },
        {
            "platform": "TadoPlatform",
            "name": "TadoPlatform",
            "homes": [
                {
                    "id": "xxxxx",
                    "name": "Zuhause",
                    "username": "xxx",
                    "password": "xxx",
                    "polling": 30,
                    "temperatureUnit": "CELSIUS",
                    "geolocation": {
                        "longitude": "xxx",
                        "latitude": "xxx"
                    },
                    "presence": {
                        "anyone": false,
                        "accTypeAnyone": "OCCUPANCY",
                        "user": [
                            {
                                "active": false,
                                "name": "iPhone xxx",
                                "accType": "OCCUPANCY"
                            },
                            {
                                "active": false,
                                "name": "iPad",
                                "accType": "OCCUPANCY"
                            }
                        ]
                    },
                    "weather": {
                        "temperatureSensor": false,
                        "solarIntensity": false,
                        "accTypeSolarIntensity": "LIGHTBULB",
                        "airQuality": false
                    },
                    "extras": {
                        "centralSwitch": false,
                        "runningInformation": false,
                        "presenceLock": false,
                        "accTypePresenceLock": "ALARM",
                        "boostSwitch": false,
                        "sheduleSwitch": false,
                        "turnoffSwitch": false,
                        "childLockSwitches": [
                            {
                                "active": false,
                                "name": "Wohnzimmer VAxxx",
                                "serialNumber": "VAxxx"
                            },
                            {
                                "active": false,
                                "name": "Schlafzimmer VAxxx",
                                "serialNumber": "VAxxx"
                            },
                            {
                                "active": false,
                                "name": "Bad VA14087xx",
                                "serialNumber": "VA1xxx"
                            }
                        ]
                    },
                    "zones": [
                        {
                            "active": true,
                            "id": 1,
                            "name": "Wohnzimmer",
                            "delaySwitch": false,
                            "autoOffDelay": false,
                            "openWindowSensor": false,
                            "openWindowSwitch": false,
                            "airQuality": false,
                            "separateTemperature": false,
                            "separateHumidity": false,
                            "minStep": "0.5",
                            "minValue": 5,
                            "maxValue": 25,
                            "mode": "MANUAL",
                            "modeTimer": 30,
                            "easyMode": false,
                            "noBattery": false,
                            "type": "HEATING",
                            "boilerTempSupport": false,
                            "accTypeBoiler": "SWITCH"
                        },
                        {
                            "active": true,
                            "id": 3,
                            "name": "Schlafzimmer",
                            "delaySwitch": false,
                            "autoOffDelay": false,
                            "openWindowSensor": false,
                            "openWindowSwitch": false,
                            "airQuality": false,
                            "separateTemperature": false,
                            "separateHumidity": false,
                            "minStep": "0.5",
                            "minValue": 5,
                            "maxValue": 25,
                            "mode": "MANUAL",
                            "modeTimer": 30,
                            "easyMode": false,
                            "noBattery": false,
                            "type": "HEATING",
                            "boilerTempSupport": false,
                            "accTypeBoiler": "SWITCH"
                        },
                        {
                            "active": true,
                            "id": 2,
                            "name": "Bad",
                            "delaySwitch": false,
                            "autoOffDelay": false,
                            "openWindowSensor": false,
                            "openWindowSwitch": false,
                            "airQuality": false,
                            "separateTemperature": false,
                            "separateHumidity": false,
                            "minStep": "0.5",
                            "minValue": 5,
                            "maxValue": 25,
                            "mode": "MANUAL",
                            "modeTimer": 30,
                            "easyMode": false,
                            "noBattery": false,
                            "type": "HEATING",
                            "boilerTempSupport": false,
                            "accTypeBoiler": "SWITCH"
                        }
                    ],
                    "telegram": {
                        "active": false
                    }
                }
            ]
        },
        {
            "name": "HomebridgeSpotifySpeaker",
            "spotifyClientId": "ClientIDxxx",
            "spotifyClientSecret": "SpotifyClientSecretxxx",
            "spotifyRedirectUri": "https://example.com/callback",
            "spotifyAuthCode": "Code",
            "spotifyPollInterval": 20,
            "deviceNotFoundRetry": {
                "enable": true,
                "retryDelay": 2000
            },
            "devices": [
                {
                    "deviceName": "Test",
                    "deviceType": "speaker",
                    "spotifyDeviceName": "xxx' 2. Echo Dot",
                    "spotifyPlaylistUrl": "https://open.spotify.com/playlist/playlisturl"
                }
            ],
            "platform": "HomebridgeSpotifySpeaker"
        }
    ],
    "disabledPlugins": []
}
```

**Screenshots:**

**Environment:**

- **Plugin Version**: v1.3.2
- **Homebridge Version**: 1.6.1
- **Node.js Version**: v18.15.0
- **NPM Version**: 
- **Operating System**: Synology NAS

**Comment by joeyhage** [comment]
Hi @epicbenz, I have published [v1.3.3](https://github.com/joeyhage/homebridge-spotify-speaker/releases/tag/v1.3.3). Please let me know if you have any issues with it.

**Comment by epicbenz** [comment]
> Hi @epicbenz, I have published [v1.3.3](https://github.com/joeyhage/homebridge-spotify-speaker/releases/tag/v1.3.3). Please let me know if you have any issues with it.

Works perfectly. Thanks a lot!
